### DISCUSSION TITLE
Esp32Cam Question ?

### DISCUSSION THREAD

**Discussion by usernameSecret** [espressif/arduino-esp32#10558] [Ideas]
Hello,

great thanks for this Project.

I started to use it, but actually I have some issus/questions.

Would it be an Idea to post a time stamp for all releases ?

Could some let me know which release of arduino esp32 was actual on around 20-29 September 2024 ?
It's because I try to reproduce a state of software set up, for an esp32cam module, which is working.
Later development of esp32cam Projects failed because of an issue, probably with deep sleep.
 
Thanks.

Sincerly,
Marc

**Comment by me-no-dev** [comment]
You can see the releases and their dates here: https://github.com/espressif/arduino-esp32/releases
